config/locales/in_person_proofing/en.yml

```diff
@@ -37,5 +37,6 @@ en:
       barcode: You’re ready to verify your identity in person
       location: Select a location to verify your ID
       state_id: Enter the information on your ID
-      verify: Please verify your information
```

Comment:
Was this properly refactored in all locations? I'm still seeing usages of `verify` on the additions side of this PR.